lib/matplotlib/axes/_axes.py

```diff
@@ -2697,7 +2697,7 @@ def get_next_color():
                       label_namer="y")
     @docstring.dedent_interpd
     def errorbar(self, x, y, yerr=None, xerr=None,
-                 fmt='', ecolor=None, elinewidth=None, capsize=None,
+                 fmt='', ecolor=None, elinewidth=None, ealpha=None, capsize=None,
```

Comment:
Somewhat annoyingly this has to go as the last argument (as if there are users passing in 8 or more positional arguments (and I would be dollars to doughnuts that there are) this will break).

One of the things I am looking forward to when we can drop python2 is that we can make everything after `fmt` keyword-only and then order them nicely.